File: SDKs/lgtm.com/v1.0/go/pkg/models/operations/gethealth.go
package operations

import (
	"openapi/pkg/models/shared"
)

type GetHealthSecurity struct {
	AccessToken shared.SchemeAccessToken `security:"scheme,type=http,subtype=bearer"`
}

type GetHealthRequest struct {
	Security GetHealthSecurity
}

type GetHealthResponse struct {
	ContentType string
	StatusCode  int64
	Health      *shared.Health
}

File: SDKs/here.com/positioning/2.1.1/typescript/src/sdk/models/operations/gethealth.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import * as shared from "../shared";



export class GetHealthResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  apiHealthStatus?: shared.ApiHealthStatus;

  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}
